### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
corrected the link for enforcing password history in Azure AD, Added steps to update password polices (only update is possible as of now) and removed the Mobile wipe statement

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2029

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -71,13 +71,18 @@ Here are some things to know about how this feature currently works as of Januar
...
-
-In addition, if an employee used a mobile device to access Microsoft 365, you can wipe it to ensure the password is no longer stored and recycled from there. To learn more, see [Wipe and block a former employee's mobile device](https://docs.microsoft.com/office365/admin/add-users/remove-former-employee?view=o365-worldwide#wipe-and-block-a-former-employees-mobile-device).
-
+If you want to prevent your users from recycling old passwords, you can do so by enforcing password history in Azure AD. See [Create a custom password policy](https://docs.microsoft.com/azure/active-directory-domain-services/password-policy#create-a-custom-password-policy).
 
 ## Synchronize user passwords hashes from an on-premises Active Directory to Azure AD (Microsoft 365)
 
 This article is for setting the expiration policy for cloud-only users (Azure AD). It doesn't apply to hybrid identity users who use password hash sync, pass-through authentication or on-premises federation like ADFS.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
This article is for setting the expiration policy for cloud-only users (Azure AD). It doesn't apply to hybrid identity users who use password hash sync, pass-through authentication, or on-premises federation like ADFS.
```

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 2]
```diff
@@ -71,13 +71,18 @@ Here are some things to know about how this feature currently works as of Januar
...
   
 To learn how to synchronize user password hashes from on premises AD to Azure AD, see [Implement password hash synchronization with Azure AD Connect sync](https://docs.microsoft.com/azure/active-directory/hybrid/how-to-connect-password-hash-synchronization).
+
+
+## Powershell to update password Policy
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
## Update password Policy
```

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 3]
```diff
@@ -71,13 +71,18 @@ Here are some things to know about how this feature currently works as of Januar
...
+
+The Set-MsolPasswordPolicy cmdlet updates the password policy of a specified domain or tenant. Two settings are required, the first is to indicate the length of time that a password remains valid before it must be changed and the second is to indicate the number of days before the password expiration date that will trigger when users will receive their first notification that their password will soon expire.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
The Set-MsolPasswordPolicy cmdlet updates the password policy of a specified domain or tenant. Two settings are required; the first is to indicate the length of time that a password remains valid before it must be changed and the second is to indicate the number of days before the password expiration date that will trigger when users will receive their first notification that their password will soon expire.
```

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 4]
```diff
@@ -71,13 +71,18 @@ Here are some things to know about how this feature currently works as of Januar
...
+
+To learn how to update password policy for a speicifc domain or tenant, See [Set-MsolPasswordPolicy](https://docs.microsoft.com/powershell/module/msonline/set-msolpasswordpolicy?view=azureadps-1.0)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
To learn how to update password policy for a specific domain or tenant, see [Set-MsolPasswordPolicy](https://docs.microsoft.com/powershell/module/msonline/set-msolpasswordpolicy?view=azureadps-1.0).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**